schemas/device/electrodeArrayUsage.schema.tpl.json

```diff
@@ -1,78 +1,49 @@
 {
   "_type": "https://openminds.ebrains.eu/ephys/ElectrodeArrayUsage",
-  "_categories": [
-    "deviceUsage"
-  ],
-  "required": [
-    "electrodeArray"
-  ],
+  "_extends": "/core/schemas/research/deviceUsage.schema.tpl.json",
   "properties": {
-    "additionalInformation": {
-      "type": "array",
-      "minItems": 1,
-      "uniqueItems": true,
-      "_instruction": "Add one or several files containing any additional information about this the electrode array.",
-      "_linkedTypes": [
-        "https://openminds.ebrains.eu/core/File",
-        "https://openminds.ebrains.eu/core/FileBundle"
-      ]
-    },
     "anatomicalLocation": {
       "type": "array",
       "minItems": 1,
       "uniqueItems": true,
-      "_instruction": "Add all anatomical entities in which this electrode array is located.",
-      "_linkedTypes": [
-        "https://openminds.ebrains.eu/controlledTerms/UBERONParcellation",
-        "https://openminds.ebrains.eu/sands/CustomAnatomicalEntity",
-        "https://openminds.ebrains.eu/sands/ParcellationEntity",
-        "https://openminds.ebrains.eu/sands/ParcellationEntityVersion"
+      "_instruction": "Add all anatomical entities that describe the semantic location of the contact point(s) of the electrode array best.",
```

Comment:
"Add all anatomical entities that semantically best describe the anatomical location of the contact point(s) of the electrode array."

Reply:
Isn't here the second part missing ? "..., in the same order that the electrode identifiers for this electrode array have been specified." or was it decided that this is the overall location of the array?

Reply:
No I think you are completely correct, but we should be careful on how we ask for it, as in some cases we only know the location of the whole array and not each of the electrodes.